PyDSS/dssInstance.py

```diff
@@ -294,41 +295,44 @@ def _UpdateControllers(self, Priority, Time, Iteration, UpdateResults):
                 elm = self._dssInstance.ActiveClass.Next()
         return maxError < self._settings.project.error_tolerance, maxError
 
-    def _CreateBusObjects(self):
-        BusNames = self._dssCircuit.AllBusNames()
-        self._dssInstance.run_command('New  Fault.DEFAULT Bus1={} enabled=no r=0.01'.format(BusNames[0]))
+    @staticmethod
+    def CreateBusObjects():
```

Comment:
Changes in the next two functions are to allow code reuse in the controllers.py file. There should not be any logic change.

Reply:
makes sense. Don't expect anything to break.